presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java

```diff
@@ -151,8 +156,18 @@ private static void checkConnectivity(CassandraSession session)
     public static void refreshSizeEstimates(String keyspace, String table)
             throws Exception
     {
-        flushTable(keyspace, table);
-        refreshSizeEstimates();
+        long deadline = System.nanoTime() + REFRESH_SIZE_ESTIMATES_TIMEOUT.roundTo(NANOSECONDS);
+        while (System.nanoTime() < deadline) {
+            flushTable(keyspace, table);
+            refreshSizeEstimates();
+            List<SizeEstimate> sizeEstimates = getSession().getSizeEstimates(keyspace, table);
+            if (!sizeEstimates.isEmpty()) {
+                log.info("Size estimates for the table %s.%s have been refreshed successfully: %s", keyspace, table, sizeEstimates);
```

Comment:
It should happen just once a 100-200 times, so it shoudn't overflood the log. I would like to be able to track it down though.